data_science_pipeline/peerscout/api/main.py

```diff
@@ -10,16 +10,35 @@
 
 LOGGER = logging.getLogger(__name__)
 
-output = {
-    "reviewing_editor_recommendation": {
-        "person_ids": [],
-        "recommendation_html": "The recommended editors are ..."
-    },
-    "senior_editor_recommendation": {
-        "person_ids": [],
-        "recommendation_html": "The recommended editors are ..."
-    }
-}
+REQUEST_JSON_SCHEMA_PATH = os.path.join(os.path.dirname(__file__), 'input-json-schema.json')
+
+
+def response_json(
+    abstract: str,
+    reviewing_editors: list,
+    senior_editors: list) -> dict:
+
+    data = {}
+
+    if abstract == "":
+        data['reviewing_editor_recommendation']={}
+        data['senior_editor_recommendation']={}
+        data['reviewing_editor_recommendation']['person_ids'] = []
+        data['senior_editor_recommendation']['person_ids'] = []
+        data['reviewing_editor_recommendation']['recommendation_html'] = "The recommended editors are ..."
+        data['senior_editor_recommendation']['recommendation_html'] = "The recommended editors are ..."
+    elif reviewing_editors == "" and senior_editors == "":
```

Comment:
I guess the editors are a list, so this should never be true?

Reply:
ah yes 👍